src/components/atoms/IconEle/index.tsx

```diff
@@ -0,0 +1,27 @@
+interface IconEleProps {
+    src: string;
+    alt: string;
+    width?: string | number;
+    height?: string | number;
+    padding?: string;
+    gap?: string;
+  }
+  
+  const IconEle: React.FC<IconEleProps> = ({
```

Comment:
please rename this component name as Icon

Reply:
This is completely deleted actually, as you can see in my local repo